### ISSUE TITLE
Argument id for data.id must not be null

### ISSUE THREAD

**Issue by omar-dulaimi** [prisma/studio#638]
```
prisma : 2.17.0
@prisma/client : 2.17.0
Current platform : debian-openssl-1.1.x
Query Engine : query-engine 3c463ebd78b1d21d8fdacdd27899e280cf686223 (at node_modules/@prisma/engines/query-engine-debian-openssl-1.1.x)
Migration Engine : migration-engine-cli 3c463ebd78b1d21d8fdacdd27899e280cf686223 (at node_modules/@prisma/engines/migration-engine-debian-openssl-1.1.x)
Introspection Engine : introspection-core 3c463ebd78b1d21d8fdacdd27899e280cf686223 (at node_modules/@prisma/engines/introspection-engine-debian-openssl-1.1.x)
Format Binary : prisma-fmt 3c463ebd78b1d21d8fdacdd27899e280cf686223 (at node_modules/@prisma/engines/prisma-fmt-debian-openssl-1.1.x)
Studio : 0.353.0
```

```
Argument id for data.id must not be null. Please use undefined instead.

Argument createdAt for data.createdAt must not be null. Please use undefined instead.
```

Prisma schema (if relevant):
```
model Ad {
  id                 String              @id @default(cuid()) @db.Char(30)
  createdAt          DateTime            @default(now())
}
```

Tested this in Electron version.

**Comment by madebysid** [comment]
@omar-dulaimi How did you get to this error? I just tried your schema on Prisma 2.17.0. I'm able to create and edit records on this model, maybe I'm missing something?

**Comment by omar-dulaimi** [comment]
@madebysid 
I'm in the process of upgrading to latest prisma coming from prisma 1.13
When I tried creating records from web studio it showed errors of not being able to stringify stuff
When I tried the same thing on the Electron version, it showed me error messages this time

The input field on Electron shows cuid as a placeholder, so I assumed it's going to generate that id on it's on, correct?

This is all the info I can provide unfortuantly.

**Comment by JMavs** [comment]
Hello,

Same here, when I try to create a new record inside a table, it says that the ID is missing. The thing is that is with autoincrement, so it should be done in an automatic way. The same behaviour occurs with datetime, uuid and everything else.

Info about my app:
- Postgres 12 using a docker image
- It was working good in the 2.15 at least (I update Prisma everytime a release is launched but I didn't use this feature)
- OS: macOS 11.2.1


https://user-images.githubusercontent.com/14231017/108976607-fd768e80-7687-11eb-8b16-15cd07d10473.mov

**Comment by omar-dulaimi** [comment]
@JMavs Thank you for providing the video to prove the existence of this issue.

**Comment by madebysid** [comment]
Hey @JMavs @omar-dulaimi I was able to reproduce this as well and I've now fixed this in Prisma CLI `2.18.0-dev.48`. If this is blocking, you should be able to `npm install prisma@dev` to get this immediately. However, I'd recommend updating to the next stable version when it comes out, since `dev` versions are mostly for internal use and may contain unannounced breaking changes.

This will also make it to stable v2.18.0 scheduled for this Tuesday!
